personality/argus/pep.pan

```diff
@@ -1,200 +1,251 @@
+  contents = contents + 'opensslSubjectAttributeIDs = http://glite.org/xacml/attribute/subject-issuer urn:oasis:names:tc:xacml:1.0:subject:subject-id' + "\n";
+  contents = contents + 'opensslSubjectAttributeDatatypes = http://www.w3.org/2001/XMLSchema#string' + "\n";
+  contents = contents + "\n";
+  contents = contents + '[GLITEXACMLPROFILE_PIP]' + "\n";
+  contents = contents + 'parserClass = org.glite.authz.pep.pip.provider.GLiteAuthorizationProfilePIPIniConfigurationParser' + "\n";
+  contents = contents + 'vomsInfoDir = ' + SITE_DEF_VOMSDIR + "\n";
+  contents = contents + 'acceptedProfileIDs = http://glite.org/xacml/profile/grid-ce/1.0 http://glite.org/xacml/profile/grid-wn/1.0' + "\n";
+  contents = contents + "\n";
+  contents = contents + '[COMMONXACMLPROFILE_PIP]' + "\n";
+  contents = contents + 'parserClass = org.glite.authz.pep.pip.provider.CommonXACMLAuthorizationProfilePIPIniConfigurationParser' + "\n";
+  contents = contents + 'vomsInfoDir = ' + SITE_DEF_VOMSDIR + "\n";
+  contents = contents + 'acceptedProfileIDs = http://dci-sec.org/xacml/profile/common-authz/1.1' + "\n";
+  contents = contents + "\n";
+  contents = contents + '#' + "\n";
+  contents = contents + '# Obligation Handlers (OH) configuration' + "\n";
+  contents = contents + '#' + "\n";
+  contents = contents + '[ACCOUNTMAPPER_OH]' + "\n";
+  contents = contents + 'parserClass = org.glite.authz.pep.obligation.dfpmap.DFPMObligationHandlerConfigurationParser' + "\n";
+  contents = contents + 'handledObligationId = http://glite.org/xacml/obligation/local-environment-map' + "\n";
+  contents = contents + 'accountMapFile = ' + SITE_DEF_GRIDMAP + "\n";
+  contents = contents + 'groupMapFile = ' + LCMAPS_CONFIG_DIR + "/groupmapfile\n";
+  contents = contents + 'gridMapDir = ' + SITE_DEF_GRIDMAPDIR + "\n";
+  contents = contents + 'useSecondaryGroupNamesForMapping = true' + "\n";
+
+  contents;
+};
+
+"/software/components/filecopy/services" =
+  npush(escape(PEP_CONFIG_FILE),
+        nlist("config",PEP_CONFIG_CONTENTS,
+              "owner","root",
+              "perms","0640",
+       )
+  );
+
+
+#-----------------------------------------------------------------------------
+# PEP system configuration file
+#-----------------------------------------------------------------------------
+
+variable PEP_SYSCONFIG_FILE ?= '/etc/sysconfig/argus-pepd';
+'/software/components/filecopy/services' = {
+    this = <<EOF;
+#
+# Options for the Argus PEP server
+#
+#JAVACMD="/usr/bin/java"
+# IPv4 instead of IPv6
+#PEPD_JOPTS="-Djava.net.preferIPv4Stack=true"
+PEPD_JOPTS=PEP_JAVA_OPTIONS
+#PEPD_HOME="/usr/share/argus/pepd"
+#PEPD_CONF="/etc/argus/pepd/pepd.ini"
+#PEPD_CONFDIR="/etc/argus/pepd"
+#PEPD_LOGDIR="/var/log/argus/pepd"
+#PEPD_LIBDIR="/var/lib/argus/pepd/lib"
+#PEPD_ENDORSEDDIR="/var/lib/argus/pepd/lib/endorsed"
+#PEPD_PROVIDEDDIR="/var/lib/argus/pepd/lib/provided"
+#PEPD_PID="/var/run/argus-pepd.pid"
```

Comment:
I just copied the original configuration file, so that the changes reflect what someone might do if doing the changes manually. I can remove them if needed.